src/analyze/tree.ts

```diff
@@ -15,7 +15,15 @@ export let orderChildrenBySize = (
 }
 
 export let accumulatePath = (root: TreeNodeInProgress, path: string, bytesInOutput: number): number => {
-  let parts = path.split('/')
+  let parts;
+  if (/^https?:\/\//.test(path)) {
+    const url = new URL(path);
```

Comment:
Should this line have a try-catch in case the user accidentally entered an invalid URL? That shouldn’t block them from looking at the visualization, right?

Reply:
Good idea! Updated the PR accordingly.